<launch>
  <node name="bicycle_can_communication" pkg="can_communication" type="Service.py" />
  <node name="bicycle_balance" pkg="bicycle" type="bicycle_balance" />
    <!-- <node pkg="bicycle" type="bicycle_balance" name="bicycle_balance" /> -->
  <!-- <node pkg="rqt_plot" type="rqt_plot" name="roll_plot" args="/Bicycle/VESC_Motor_1_info/Motor_Roll">   
    <param name="geometry" value="600x400" />
  </node>
  <node pkg="rqt_plot" type="rqt_plot" name="rpm_plot" args="/Bicycle/VESC_Motor_1_info/Motor_rpm">
      <param name="geometry" value="600x400" />
  </node>
  <node pkg="rqt_plot" type="rqt_plot" name="roll_gyro_plot" args="/Bicycle/VESC_Motor_1_info/Motor_Roll_Gyro">
      <param name="geometry" value="600x400" />
  </node> -->
  <node pkg="fdilink_ahrs" name="ahrs_driver" type="ahrs_driver" output="screen" >
    <param name="debug"  value="false"/>
    <param name="port"  value="/dev/ttyUSB0"/>
    <param name="baud"  value="921600"/>
    <param name="imu_topic"  value="/fdi_imu"/>
    <param name="imu_frame"  value="imu_link"/>
    <param name="device_type"  value="1"/> <!-- 0: origin_data, 1: for single imu or ucar in ROS, 2:for Xiao in ROS -->
  </node>

  <!-- <node pkg="dynamic_reconfigure" type="dynamic_reconfigure" name="my_node">
    <param name="config" type="string" value="DynamicLQR.cfg" />
  </node> -->
</launch>
